xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd

```diff
@@ -0,0 +1,357 @@
+				<xsd:sequence>
+					<xsd:group ref="SeatEquipmentGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="SeatEquipmentGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for SEAT EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="SeatBackHeight" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Height of Seat back from seat bottom.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="SeatDepth" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Depth of Seat from front to back.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="IsFoldup" type="xsd:boolean" default="false" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether seat folds up. default is false</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="IsReclining" type="xsd:boolean" default="false" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether seat reclines. Default is false.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- === BED EQUIPMENT ============================================ -->
+	<xsd:element name="BedEquipment" substitutionGroup="InstalledEquipment">
+		<xsd:annotation>
+			<xsd:documentation>A specialisation of SPOT EQUIPMENT describing the detailed properties of a bed. +v2.0</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="BedEquipment_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EquipmentGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="SpotEquipmentGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="BedEquipmentGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="BedEquipmentIdType" use="optional">
+						<xsd:annotation>
+							<xsd:documentation>Identifier of ENTITY.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="BedEquipment_VersionStructure" abstract="false">
+		<xsd:annotation>
+			<xsd:documentation>Type for a BED EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="SpotEquipment_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="BedEquipmentGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="BedEquipmentGroup">
```

Comment:
Shouldn't we have a "Level" (ground, one up, and maybe two up) and maybe some info on the climbing device (but that's probably always a ladder) and space above ?

Reply:
YEs we could add BunkLevel   bottom/first/top l and shoudl also add headroom